code/game/antagonist/antagonist_factions.dm

```diff
@@ -1,11 +1,11 @@
 /mob/living/proc/convert_to_rev(mob/M as mob in oview(src))
-	set name = "Convert Bourgeoise"
+	set name = "Invite to the Contendors"
```

Comment:
Contendor isn't an actual word, did you mean contenders?

Reply:
Yup it's a typo, contendor is like, a word in spanish but definitely not english. Lemme fix

Reply:
Alright this should be fixed now.